deploy/operator_ironic.yaml

```diff
@@ -14,6 +14,56 @@ spec:
     spec:
       serviceAccountName: metal3-baremetal-operator
       hostNetwork: true
+      initContainers:
+        - name: ipa-downloader
+          image: quay.io/metal3-io/ironic-ipa-downloader:master
+          command:
+            - /usr/local/bin/get-resource.sh
+          imagePullPolicy: Always
+          securityContext:
+            privileged: true
+          env:
+            - name: CACHEURL
+              valueFrom:
+                configMapKeyRef:
+                  name: ironic-bmo-configmap
+                  key: cache_url
+        - name: rhcos-downloader
```

Comment:
is this something you want addressed now?  Honestly not sure how we'd do this another way atm.

Reply:
Yes, I think this should be removed from this PR since it's OpenShift specific.

It should go somewhere where details of the OpenShift integration are maintained.  That's openshift-metal3/dev-scripts short term, and the machine-api-operator longer term (but hopefully ASAP)

Reply:
Maybe we should just copy this entire yaml to dev-scripts until the MAO change lands?

Reply:
/cc @sadasu